### PR TITLE
Client server db setup

### PR DESCRIPTION
**Pull request by Maksyym**
Changes:
-Client displaying temporary React placeholder (static files served correctly)
-Server API route correctly responds with Room Data
-Schema working as intended, seed script written

Next Steps:
-Flush out roomImg DB table
-Figure out proper querying/helper functions to retrieve neatly formatted data, linking many images per room

Future Steps:
-Set up ESLint
-Set up Jest testing
-Set up Enzyme testing
-Set up CircleCI

### CODE REVIEW

File: public/bundle.js [hunk 1]
```diff
@@ -81,19 +81,19 @@
 /******/
 /******/
 /******/ 	// Load entry module and return exports
-/******/ 	return __webpack_require__(__webpack_require__.s = "./client/src/index.jsx");
+/******/ 	return __webpack_require__(__webpack_require__.s = "./client/index.jsx");
```
Comment (eggtart26) [inline public/bundle.js]: may be you should 'public/bundle.js' into your .gitignore file

**Review by bregbromley — COMMENTED** [review]
Looks good! my name is breg bromley and i approve of this message

**Comment by eggtart26** [comment]
looks good, it might be better if put the 'public/bundle.js' into your .gitignore file

**Comment by Maksyym** [comment]
Implemented Justin's critique, ready to merge!

**Pull request merged**